pages/common/mosquitto_sub.md

```diff
@@ -0,0 +1,19 @@
+# mosquitto_sub
+
+> A simple MQTT version 3.1.1 client that will subscribe to topics and print the messages that it receives.
+
+- Subscribe to all broker status messages:
+
+`mosquitto_sub -v -t {{\$SYS/#}}`
+
+- Subscribe to "topic" sensors/temperature information on `localhost` with `QoS` 1:
```

Comment:
Perhaps this will read better?

```suggestion
- Subscribe to the topic `sensors/temperature` information on `localhost` with `QoS` 1:
```

Also, what does `QoS` stand for? Could we expand it at all if it's an abbreviation?

Reply:
`QoS` stands for Quality of Service.
Will update the doc